docs/language/operators/summarize.md

```diff
@@ -34,7 +37,16 @@ to a cluster of workers in an adaptive shuffle, though this is not yet implement
 
 ### Examples
 
-Sum the input sequence:
+Average the input sequence:
+```mdtest-command
+echo '1 2 3 4' | zq -z 'summarize avg(this)' -
```

Comment:
Once again, I chose a different aggregate function so we'd not be _leading_ with `summarize sum` and perhaps confuse new users.